environment-dev.yml

```diff
@@ -6,8 +6,8 @@ dependencies:
   - numpy>=1.16.5
   - pyarrow>=1.0.0
   - libthrift<0.14.1  # ref: modin gh-2840
-  - dask[complete]>=2.12.0,<=2.19.0
-  - distributed>=2.12.0,<=2.19.0
+  - dask[complete]>=2.22.0
```

Comment:
```suggestion
  - dask[complete]>=2020
```

Reply:
As we've removed the upper pin, it now installs whatever is recent enough, doesn't it? I see `dask 2021.6.2` being used right now in CI.